product-service/src/functions/createProduct/index.ts

```diff
@@ -0,0 +1,46 @@
+import { handlerPath } from '@libs/handler-resolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  role: 'arn:aws:iam::974074737183:role/DynamoDbLambdaAccessRole',
```

Comment:
**Suggestion**: Better to store in serverless.config

Reply:
not sure about that, what if in the future I will have lambdas that shouldn't have access to dynamodb

Reply:
I guess it could be solved in config.